src/Command/CommandHelper.php

```diff
@@ -310,7 +310,12 @@ public static function begin(
 					require_once $file;
 				})($bootstrapFile);
 			} catch (\Throwable $e) {
-				$errorOutput->writeLineFormatted($e->getMessage());
+				$errorOutput->writeLineFormatted('Exception thrown in ' . $e->getFile() . ' on line ' . $e->getLine() . ': ' . $e->getMessage() . ' while loading Bootstrap file ' . $bootstrapFile);
+
+				if ((bool) $input->getOption('debug')) {
```

Comment:
This will crash in commands that don't have the `--debug` flag. Add a `$debug` parameter to `CommandHelper::begin()` and pass it explicitly in all places where it's called.